### DISCUSSION TITLE
Name of the very top bar

### DISCUSSION THREAD

**Discussion by progrium** [microsoft/vscode-discussions#2390] [Extension Development QnA] [answered]
In the diagram on [UX Guidelines](https://code.visualstudio.com/api/ux-guidelines/overview), I don't see this bar of the very top including the top right layout buttons. I'm curious the name of this UI element and whether items can be contributed to it:

<img width="473" alt="Screenshot 2024-11-08 at 2 19 15 PM" src="https://github.com/user-attachments/assets/91cd7369-3eea-4f59-b11e-9be1988d6043">

**Comment by alefragnani** [comment, marked as answer]
Hi @progrium ,

It is called `Title Bar`, as used here while describing the Secondary Side Bar customization  (https://code.visualstudio.com/docs/editor/custom-layout#_secondary-side-bar).

About contributions, no, there is no support for that. And running a quick search in the VS Code repo, I didn’t find any open feature request about it either.

Hope this helps

**Reply by robconery** [reply]
@progrium does this answer your question?
